lib/handlers.py

```diff
@@ -439,6 +438,12 @@ def _save_ef_per_m_nc4(
                 if name not in req_coords:
                     ds = ds.drop_vars(name)
 
+            # add `forecast_reference_time` as a coordinate sidecar to `time`
+            dt_frt = datetime.fromtimestamp(job.model_run_at, tz=UTC).replace(
+                tzinfo=None
+            )
+            ds = ds.assign_coords(forecast_reference_time=("time", np.array([dt_frt])))
```

Comment:
@zebengberg can confirm here, but I believe you'll want to do:

```suggestion
            ds = ds.assign_coords(forecast_reference_time=("time", np.array([dt_frt], dtype='datetime64')))
```

Otherwise you'll get something out that is like `np.array([datetime.datetime(...)], dtype='object')`

Its likely that xarray would fix this, but best to be explicity

Reply:
Yes your suggestion looks right to me.

Reply:
👍 I'll make that change. Agreed better to be explicit.

FWIW, I did test this and it does treat the field as a `datetime64[ns]` type, but interestingly only if `dt_frt` is a timezone naive `datetime.datetime` object (hence ln. 442).  When I tried leaving it as a timezone aware `datetime.datetime` with `tz=UTC`, it does indeed represent it as `object` in the xarray `Dataset`.